posts/2023-11-05-telegram-bot-for-file-sharing-using-quarkus.adoc

```diff
@@ -0,0 +1,540 @@
+= Telegram bot for file sharing using Quarkus
+https://github.com/iarmush
```

Comment:
I don't know what is the exact rule here, or are you trying to achieve, but the idea is to put your github username here and then it will fetch you real name and avatar - meaning just `iarmush`